config/database.yml

```diff
@@ -17,16 +17,17 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  database: netjam
+  username: nj
+  password: accessgood
```

Comment:
no because these are dev credentials. when this thing is actually running in production mode we will give it env vars so random github people won't know our shit